### PR TITLE
Redirect slashless static sites to the slashful version

### PR DESCRIPTION
**Pull request by yuvipanda**
slash slash slashes slash slash

Fixes https://github.com/2i2c-org/infrastructure/issues/1517

### CODE REVIEW

File: helm-charts/basehub/templates/static/configmap.yaml [hunk 1]
```diff
@@ -9,9 +9,15 @@ data:
   nginx.conf: |
     server {
       listen 8080;
-      location {{ .Values.staticWebsite.ingress.path }} {
+      # We require path to *not* have a trailing slash
```
Comment (consideRatio) [inline helm-charts/basehub/templates/static/configmap.yaml:12]:
```suggestion
      # We enforce path to *not* have a trailing slash via the helm chart schema
```

**Review by consideRatio — APPROVED** [review]
Great to see an example on how this is done!!

**Review by GeorgianaElena — APPROVED** [review]
🚀

**Comment by yuvipanda** [comment]
I love writing nginx config! Been so long...

**Comment by yuvipanda** [comment]
fun fact - https://github.com/yuvipanda/jupyterhub-nginx-chp was one of my first JupyterHub projects, that was dynamically doing nginx config as a replacement for configurable-http-proxy

**Pull request merged**